### ISSUE TITLE
Expection running /crate util item on items after 1.15 update

### ISSUE THREAD

**Issue by samerbam** [Hazebyte/CrateReloaded#516]
### How to reproduce:
* ~~Pickup bee hive with any amount of bees in it (Tested with 3)~~
* ~~hold bee hive and run `/crate util item`~~
* Run `/crate util item` on any item from after the 1.15 update

### Console Log
```
19:35:56 <redacted username> issued server command: /cr util item
--
  |  
  | 19:35:56 ERROR
  | [CrateReloaded]: An error has occurred. Please report this to the author...
  |  
  | 19:35:56 ItemStack{BEE_NEST x 1, TILE_ENTITY_META:{meta-type=TILE_ENTITY, BlockStateTag={honey_level=5i}, internal=H4sIAAAAAAAAAO1WbWhbVRg+NzfJmvRr/fBjw41WBBGk4oY/qra5afqRYNqmH1nnhtbTe0+TQ27OuTv3pF3dH0WR6aB0foLbD2WItGUdbtUyFCsTf2wo4sePQVEUFXQOQSw659b63txkkzkGg63905Bwb3Lf5z3P+7zv+5AgQkFU0WJyPdPGJJVj/TgVQN4WQuwgQkgNoqD7eyuWuARVJbBFREOfhUdZL8E2Z6i0LxEe6GrrHWzr6PCgQCvBMt1PswTAqLoll8lQ2ZCzDCxJnIwQE5J6FFTSzToEzzEDKSgYJ8MyiplBDORD68NDNheWpJyFsxAhAYACKBiWUtChnCzwKkHeLgyH1GUpI7rAw/LBFGFEUL0hy0dIljA5aFuEGH4oBtsk9NXWrZqTqgi84ypAvHswTbAp0wWQdhdyX0XQ5v+Dhrlp8tFBgVmKBFCgkxt0mBKRZ6kUcdW9UB7P1tmObnVDnOVsP/K79YVOVC429kx9rKJAN2iLncodbCnyJpOxVrj17l/3+QOJmvDF979Z2vbCXPtjqMgvWuCnoEA4ReLQROIoWhZjIzkT+OEhk0B/fS0CUxZEJVmS5YIS21G0zG1lt6ApyvyOptr4hXw2LfmRe9Ut5Kq/PiwA2Cq4FUljphPb59Bqmqnd89+PilRg4SJKernMl+JEerTG+Tec31VUGc0J2TLmDIgtcdZyoyud9l+R3lNMC/UUxig8igVRYHqi2O6TOZaChIF2LnRiFA72ovIBLkzDUa6T2/KlXxdPsXMd30JcGBok3MF0B8opKSZJNt8qr0vvln6qZ+w+CiQS0FjKiu0ARMAheQngcd5qmArl3iCQgBkgIsFtFSk74OnrcN0OCeFg5VH4PgBdWBelI+RyyMErQrYDh4pL5OOwXPIAf+qgtnfyrAcqBtlc8goqj2CWAJ5JK865dJoPenQRXWKhqKiqz6IpLhukU0kcjjQQ2jCjoo0R0NeGrRtlV6vPh8rasWm2UmgLPC1MVRfvEHgE1h9G2UMNVH55AYYIzLu/kzsJ8uMz/8xcbGLfKzOh49XWO0/ethiaat605Wx2yYO87VSQ5WUV1QJzxiU4ChGOGnm1XXVVUKYwhYs9e+ufO6Bt807F7jsypuk9y498ueVuFfljLA677VILOmzbzTEKU+BD/mh+b8N1zqOaBOwf1EGgjl6yKweHGyBMRYILic0I56YBIiC349WdlHXres7KK1EgFHhCRaX5+xhzeIIDzKM1K7zpVvjW3E7298ufXsMKvx/9Tmm6GBmf/eK3nRMLD4+vmBUmbpAVRo6eP7ZmhatnhZs2rIAVhvbfWX9+WP8ndGTj6cfrPZ+EJk8s7X73zO3XbYV//Hjyp68PacnfN88eentB05PJNzNVTatrhT6yZoU33wp3/Txl3d9Ycw0r/ED75dXs87FzLy6ka5Wjs/esmBVO3CgrPBZ8b80KV88K66ZX4l/hsw+lE82+z0Iz02dqx+3XQoenqw+f/Ov49Vvh9I5b90xpyT8voMy+U5re/8Pp5g+fXl0rVCYh+l+TWSLMQw8AAA==, blockMaterial=BEE_NEST}}
  |  
  | 19:35:56 WARN
  | [CrateReloaded]: Error occured while executing command crate
  |  
  | 19:35:56 ERROR
  | [CrateReloaded]: [ACF] Exception in command: crate util item
  | [CrateReloaded]: [ACF] java.lang.reflect.InvocationTargetException
  | [CrateReloaded]: [ACF] at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  | [CrateReloaded]: [ACF] at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  | [CrateReloaded]: [ACF] at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  | [CrateReloaded]: [ACF] at java.base/java.lang.reflect.Method.invoke(Method.java:566)
  | [CrateReloaded]: [ACF] at com.hazebyte.acf.RegisteredCommand.invoke(RegisteredCommand.java:154)
  | [CrateReloaded]: [ACF] at com.hazebyte.acf.BaseCommand.executeCommand(BaseCommand.java:577)
  | [CrateReloaded]: [ACF] at com.hazebyte.acf.BaseCommand.execute(BaseCommand.java:513)
  | [CrateReloaded]: [ACF] at com.hazebyte.acf.RootCommand.execute(RootCommand.java:99)
  | [CrateReloaded]: [ACF] at com.hazebyte.acf.BukkitRootCommand.execute(BukkitRootCommand.java:81)
  | [CrateReloaded]: [ACF] at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159)
  | [CrateReloaded]: [ACF] at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:807)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.PlayerConnection.handleCommand(PlayerConnection.java:2021)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.PlayerConnection.c(PlayerConnection.java:1832)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:1785)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.PacketPlayInChat.a(PacketPlayInChat.java:47)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.PacketPlayInChat.a(PacketPlayInChat.java:5)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:23)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1132)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1125)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1086)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1000)
  | [CrateReloaded]: [ACF] at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:173)
  | [CrateReloaded]: [ACF] at java.base/java.lang.Thread.run(Thread.java:834)
  | [CrateReloaded]: [ACF] Caused by: java.lang.NullPointerException
  | [CrateReloaded]: [ACF] at com.hazebyte.crate.api.util.Messenger.tell(Messenger.java:81)
  | [CrateReloaded]: [ACF] at com.hazebyte.crate.api.util.Messenger.tell(Messenger.java:45)
  | [CrateReloaded]: [ACF] at com.hazebyte.crate.api.util.Messenger.tell(Messenger.java:60)
  | [CrateReloaded]: [ACF] at cratereloaded.m$b.i(CrateCommand.java:118)
  | [CrateReloaded]: [ACF] ... 28 more
```

**Comment by samerbam** [comment]
### Update: 
Seems like the exception occurs whenever you run `/crate util item` on any item that was added after the 1.15 update

Known Items:
* bee_nest
* beehive
* bee_spawn_egg
* honey_bottle
* soul_campfire
* soul_lantern
* All Netherite items

**Comment by imWillX** [comment]
Fixed in `2.0.38`
